GradleInfo reader assigns JAVA to groovy

Hi, 

I'm trying to use Dekorate in version 1.0.1 using gradle as build tool. The manifests are generated:
```
> Task :compileJava FAILED
[WARN] Could not detect project version. Using 'latest'.
[INFO] Found @KubernetesApplication on: example.micronaut.Application
[INFO] Initializing dekorate session.
[INFO] Generating manifests.
[INFO] Processing kubernetes configuration.
[INFO] Writing: file:/private/var/folders/_q/qs930dw93b173jy20p05b3x00000gn/T/dekorate6190129776997829285/build/classes/java/main/META-INF/dekorate/kubernetes.json
[INFO] Writing: file:/private/var/folders/_q/qs930dw93b173jy20p05b3x00000gn/T/dekorate6190129776997829285/build/classes/java/main/META-INF/dekorate/kubernetes.yml

```

However the build on 
```
Caused by: java.lang.RuntimeException: Error writing resources
        at io.dekorate.processor.SimpleFileWriter.write(SimpleFileWriter.java:89)
        at io.dekorate.processor.AptWriter.lambda$write$1(AptWriter.java:69)
        at io.dekorate.processor.AptWriter.write(AptWriter.java:69)
        at io.dekorate.Session.close(Session.java:207)
        at io.dekorate.kubernetes.apt.KubernetesAnnotationProcessor.process(KubernetesAnnotationProcessor.java:42)
        at org.gradle.api.internal.tasks.compile.processing.DelegatingProcessor.process(DelegatingProcessor.java:62)
        at org.gradle.api.internal.tasks.compile.processing.NonIncrementalProcessor.process(NonIncrementalProcessor.java:45)
        at org.gradle.api.internal.tasks.compile.processing.DelegatingProcessor.process(DelegatingProcessor.java:62)
        at org.gradle.api.internal.tasks.compile.processing.TimeTrackingProcessor.access$401(TimeTrackingProcessor.java:37)
        at org.gradle.api.internal.tasks.compile.processing.TimeTrackingProcessor$5.create(TimeTrackingProcessor.java:99)
        at org.gradle.api.internal.tasks.compile.processing.TimeTrackingProcessor$5.create(TimeTrackingProcessor.java:96)
        at org.gradle.api.internal.tasks.compile.processing.TimeTrackingProcessor.track(TimeTrackingProcessor.java:117)
        at org.gradle.api.internal.tasks.compile.processing.TimeTrackingProcessor.process(TimeTrackingProcessor.java:96)
        at com.sun.tools.javac.processing.JavacProcessingEnvironment.callProcessor(JavacProcessingEnvironment.java:802)
        at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$200(JavacProcessingEnvironment.java:91)
        at com.sun.tools.javac.processing.JavacProcessingEnvironment$DiscoveredProcessors$ProcessorStateIterator.runContributingProcs(JavacProcessingEnvironment.java:635)
        at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1041)
        at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1206)
        at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1170)
        at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:856)
        at com.sun.tools.javac.main.Main.compile(Main.java:523)
        ... 122 more
Caused by: java.io.FileNotFoundException: /private/var/folders/_q/qs930dw93b173jy20p05b3x00000gn/T/dekorate6190129776997829285/build/classes/groovy/main/../.dekorate/config/docker.yml (No such file or directory)
        at io.dekorate.processor.SimpleFileWriter.write(SimpleFileWriter.java:81)
        ... 142 more

```

Obviously Dekorate tries to operate with files on path `/build/classes/groovy/main`, interesting fact is that  the k8s manifests were generated into `/build/classes/java/main/`. 

Anyway, for version 1.0.0, it works. There are some commits in [1.0.0...1.0.1](https://github.com/dekorateio/dekorate/compare/1.0.0...1.0.1) that could introduce such regression.

--- a/core/src/main/java/io/dekorate/project/GradleInfoReader.java
+++ b/core/src/main/java/io/dekorate/project/GradleInfoReader.java
@@ -53,7 +53,8 @@ public class GradleInfoReader implements BuildInfoReader {
 
   private static final String CLASSES = "classes";
   private static final String GROOVY = "groovy";
-  private static final String JAVA = "groovy";
+  private static final String JAVA = "java";
+  private static final String KOTLIN = "kotlin";
 
   private static final String OPEN_BRACKET = "{";
   private static final String CLOSE_BRACKET = "}";
@@ -79,6 +80,17 @@ public class GradleInfoReader implements BuildInfoReader {
     return root.resolve(BUILD_GRADLE_GROOVY).toFile().exists() || root.resolve(BUILD_GRADLE_KTS).toFile().exists();
   }
 
+  private String detectLanguage(Path root) {
+    Path buildDir = root.resolve(BUILD).resolve(CLASSES);
+    if (buildDir.resolve(KOTLIN).toFile().isDirectory()) {
+      return KOTLIN;
+    } else if (buildDir.resolve(GROOVY).toFile().isDirectory()) {
+      return GROOVY;
+    } else {
+      return JAVA;
+    }
+  }
+
   @Override
   public BuildInfo getInfo(Path root) {
     Path buildGradle = root.resolve(BUILD_GRADLE_GROOVY);
@@ -88,6 +100,8 @@ public class GradleInfoReader implements BuildInfoReader {
        buildGradle = root.resolve(BUILD_GRADLE_KTS);
        settingsGradle = root.resolve(SETTINGS_GRADLE_KTS);
     }
+
+    String language = detectLanguage(root);
 
     Path gradleProperties = root.resolve(GRADLE_PROPERTIES);
 
@@ -127,7 +141,7 @@ public class GradleInfoReader implements BuildInfoReader {
       .withBuildTool(GRADLE)
       .withBuildToolVersion(Gradle.getVersion(root))
       .withOutputFile(outputDir.resolve(sb.toString()))
-      .withClassOutputDir(root.resolve(BUILD).resolve(CLASSES).resolve(JAVA).resolve(MAIN))
+      .withClassOutputDir(root.resolve(BUILD).resolve(CLASSES).resolve(language).resolve(MAIN))
       .withResourceDir(root.resolve(SRC).resolve(MAIN).resolve(RESOURCES))
       .build();
   }

